requirements-minimal.txt

```
@@ -19,3 +19,7 @@ signalfx
 simplejson
 sortedcontainers
 typing-extensions
+rook==0.1.160
+distro==1.7.0
+funcsigs==1.0.2
+psutil==5.9.1
```

```suggestion
rook
```

this file should only include libraries that are directly imported in production code